### DISCUSSION TITLE
Zettlr and Android

### DISCUSSION THREAD

**Discussion by BabakFarshchian** [Zettlr/Zettlr#4826] [Show and tell]
No I did not port Zettlr to Android! But I wanted to share a tip about using the Android app called Zettel Notes together with Zettlr. I even wrote a blog post about it: https://blog.babak.no/2023/02/04/how-to-almost-use-zettlr-on-android-devices/

No endorsement. I am not related to the developers of none of the apps.

Cheers, and let us know how you do it.

**Comment by skoufaki** [comment]
Unfortunately it's not FLOSS (free libre open source), but proprietary. :(

https://github.com/damionx7/Zettel-Notes-Documentation/issues/1
